setup.py

```diff
@@ -134,19 +143,23 @@ def vitables_data_files():
 # ========== #
 # main stuff #
 # ========== #
-
-setup(name="liam2",
-      # cx_freeze wants only ints and dots
-      version=int_version('0.10.2'),
-      description="LIAM2",
-      cmdclass={"build_ext": MyBuildExt},
-      ext_modules=ext_modules,
-      options={"build_ext": build_ext_options, "build_exe": build_exe_options},
-      executables=[Executable("main.py")],
-      install_requires=['numpy', 'numexpr', 'tables'],
-      extras_require=dict(
-          interpolation=['bcolz'],
-          plot=['matplotlib'],
-          view=['vitables'],
-          ),
-      )
+other_kwargs = dict()
+if Executable is not None:
+    other_kwargs['executables'] = [Executable("main.py")]

```

Comment:
Not on windows and I do pip install on my debian machine. Do you have time to give it a try ?